### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -49,6 +49,66 @@ There's nothing that you have to do. If you've added the newest branch under `JO
...
+The Gradle build comes with some useful tasks (run `./gradlew tasks` to see the whole list). Let's focus on
+the following ones
+
+```
+Jenkins tasks
+-------------
+startJenkins - Starts Jenkins via Docker Compose
+stopJenkins - Stops Jenkins via Docker Compose
+
+SCST tasks
```
Comment (sabbyanandan) [inline README.adoc]: Let's the header in full-form i.e., "Spring Cloud Stream Tasks". FYI - we usually refer Spring Cloud Stream as `SCSt`.

File: README.adoc [hunk 2]
```diff
@@ -49,6 +49,66 @@ There's nothing that you have to do. If you've added the newest branch under `JO
...
+----------
+initializeAndStartScst - Removes unnecessary code for SCST and starts Jenkins
```
Comment (sabbyanandan) [inline README.adoc]: SCST should be `SCSt`; also, `initializeAndStartScst` could be `initializeAndStartSCSt` for consistency.

**Review by sabbyanandan — COMMENTED** [review]
LGTM

File: README.adoc (hunk 2 above)
Comment (marcingrzejszczak) [reply README.adoc]: It's camel case. Let's maybe not exaggerate on this one? :P

File: README.adoc (hunk 1 above)
Comment (marcingrzejszczak) [reply README.adoc]: Sure!

**Comment by marcingrzejszczak** [comment]
Done in https://github.com/spring-io/build-scripts/commit/4eaf9b4ee767222b2e25a03d5358d90324fff816

**Pull request closed without merging**